src/Shop/FunctionalCalculatorService2.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace DDDHH\Shop;
+
+use DDDHH\Shop\Cart\Cart;
+use DDDHH\Shop\Cart\Item;
+
+class FunctionalCalculatorService2 implements CalculatorService
```

Comment:
What is the point of this class? It contains an error in the calculation so why not name it `BrokenFunctionalCalculatorService` or something to make its faultiness obvious.

Then again, I miss tests.